src/components/organisms/OcResource/OcResource.vue

```diff
@@ -43,8 +43,21 @@
         :full-path="resource.path"
         :is-path-displayed="isPathDisplayed"
       />
-      <div v-if="resource.indicators.length > 0" class="oc-resource-indicators">
-        <oc-status-indicators :resource="resource" :indicators="resource.indicators" />
+      <div class="oc-resource-indicators">
+        <router-link
```

Comment:
is the resource path guaranteed to be always a link? What if we just want to use it to transport some meta information without having it routable?